content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -79,42 +79,62 @@ With Istio, you can enable authentication for end users through [request authent
     <redacted>
     {{< /text >}}
 
-## Authorization is too restrictive
-
-When you first enable authorization for a service, all requests are denied by default. After you add one or more authorization policies, then
-matching requests should flow through. If all requests continue to be denied, you can try the following:
+## Authorization is too restrictive or permissive
+
+### Make sure there are no typo in the policy YAML file
```

Comment:
Yaml is hard :slightly_smiling_face:

Reply:
More seriously though - kubernetes has `describe` which could possible help understand these. Do we need/want something similar?

Reply:
yeah, I think we considered similar things in `istioctl analyze` before just haven't had time to work on this.